### DISCUSSION TITLE
OpenDTU ohne Internet

### DISCUSSION THREAD

**Discussion by martin-krieger** [tbnobody/OpenDTU#810] [General]
Ich möchte eine Kleinsolaranlage an einem Ort ohne Internetzugang errichten. Alle Beschreibungen von OpenDtu (und Ahoy) gehen von der Einbindung in ein Wlan aus. Ist es möglich (und sinnvoll) so eine DTU auch eigenständig, ohne Netzzugang, zu betreiben? Also (mit einem Tablet oder Smartphone) direkt über den DTU-AP darauf zuzugreifen und die Daten auszulesen? Und sie dann gerne auch auf einem Kleindisplay anzuzeigen?

**Comment by Solarteur** [comment]
Ähnliches wurde hier
https://github.com/tbnobody/OpenDTU/discussions/601
schon einmal (an)diskutiert.

**Reply by martin-krieger** [reply]
Ah ja, danke. Das Problem scheint demnach die fehlende aktuelle Zeit zu sein. Aber wozu wird die denn überhaupt benötigt? Die aktuellen Daten (Strom, Spannung etc.) sind doch unabhängig davon auf "jetzt" bezogen.
